tests/model_geometry_test.js

```diff
@@ -0,0 +1,486 @@
+'use strict';
+
+require('./helper');
+
+const { expect } = require('chai'),
+  { DataTypes } = require('../source'),
+  dialect = 'postgres',
+  semver = require('semver');
+
+const version_helper = require('../source/version_helper.js')
+const { makeTestSequelizeInstance } = require('./helper.js')
+
+function maybeSkip(test, isCRDBVersion21_2Plus) {
+  if (!isCRDBVersion21_2Plus) {
+    test.skip()
+  }
+}
+
+// Edited test:
+// It is expected to have CRS field in GEOMETRY fields.
+// Geometry is only supported in versions 21.2+. We only run this if 
+// we're on a version of CockroachDB equal or greater to 21.1.
```

Comment:
the skips all seem fine, but i think the last word of this sentence should be `21.2` right?